src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
+	<a href="https://www.facebook.com/groups/1038072369705184">
+		<img class="social-media" src={fb_logo} alt="Facebook logo" />
+	</a>
+	<a href="https://www.instagram.com/kendoumich/">
```

Comment:
I feel like we should display the handle on the screen.

Reply:
If we do that, then what's a good way to style it? One option is to go back to using figure and use <figcaption>. Another way is to use a div and center the caption under the image. <figcaption> might be better for screen readers.

Reply:
No I really didn't think this is a figure. It's not supplemental to the main content. It is the content. It's up to you, we could put the image inline with the handle or on the next line. Maybe look around for some examples.

Reply:
Alright. For now I'm going to add the handles in a really simple but messy/ugly way. I already have an idea for how to do it better, and I'll do that in #21 in order to keep separation between this issue and that one.
Added in [08d8d0a](https://github.com/kendoclub-at-umich/website/pull/32/commits/08d8d0a3980188c4195a9938b0126023a2546a3c)

Reply:
After conversation, we decided to relocate/move some of the social media. Resolved in [5e58e97](https://github.com/kendoclub-at-umich/website/pull/32/commits/5e58e970d6d50441610fd35db7b9caf3c7f235e7)